install-testnet-relay.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+DONE_START='\n\e[1;32m' # new line + bold + green
+DONE_END='\n\n\e[0m'    # new line + reset
+
+WORKDIR=$PWD
+
+printf "${LOG_START}Initializing relay...${LOG_END}"
+
+cd "$WORKDIR/tbtc/solidity"
+
+# Get network ID.
+NETWORK_ID_OUTPUT=$(truffle exec ./scripts/get-network-id.js)
+NETWORK_ID=$(echo "$NETWORK_ID_OUTPUT" | tail -1)
+
+# Extract TestnetRelay contract address.
+JSON_QUERY=".networks.\"${NETWORK_ID}\".address"
+TESTNET_RELAY_CONTRACT="$WORKDIR/tbtc/solidity/build/contracts/TestnetRelay.json"
+TESTNET_RELAY_CONTRACT_ADDRESS=$(cat ${TESTNET_RELAY_CONTRACT} | jq "${JSON_QUERY}" | tr -d '"')
+
+printf "${LOG_START}TestnetRelay contract address is: ${TESTNET_RELAY_CONTRACT_ADDRESS}${LOG_END}"
+
+cd $WORKDIR
+
+# Copy all config files to the right relay directory.
+cp -R configs/relays/. relays/maintainer/maintainer/config/
+
+cd "$WORKDIR/relays/maintainer/maintainer/config"
+
+# Fill SUMMA_RELAY_CONTRACT env in config file with their actual value.
+sed -i .OLD 's:RELAYCONTRACT:'$TESTNET_RELAY_CONTRACT_ADDRESS':' .my_env_file.env
+rm .my_env_file.env.OLD
+
+cd $WORKDIR
+
+# Install right python version to run the relay.
+pyenv install 3.7.0
```

Comment:
Is `3.7.0` a hard requirement? Looks like the latest available `3.7.x` is `3.7.7`.

Reply:
Yeah we can use `3.7.7`